devmanual.css

```diff
@@ -1,3 +1,13 @@
+.logo {
+    display: inline;
+}
+
+@media only screen and (min-width: 380px) {
+    .logo {
+        display: inline-block;
+    }
+}
+
```

Comment:
Sorry for the late reply.

I don't see any effect of this in my tests. devmanual.css is loaded before tyrian.min.css, and IIUC the latter overrides the former by setting `display: inline-block;` unconditionally for the `.logo`.

What am I missing here?

Reply:
Looks like I got this one wrong. If you navigate to https://devmanual.gentoo.org with Responsive Design Mode set to 320 x 480 in Firefox Developer tools, you will see that `display: inline-block` is causing the image to shift off screen. By setting it to `display: inherit`, the logo then bumps back up to the proper location. I was attempting to only set inline-block if the page width is 380px or greater. Otherwise it should be inherited.

Reply:
I still see no difference, even if I change `inline` to `inherit` in your patch. AFAICS it has no effect bacause it is superseded by the `.logo` block in the Tyrian layout.

Reply:
Yes, I think it is in the wrong order.